wbooks/src/redux/Books/reducer.js

```diff
@@ -0,0 +1,33 @@
+import {ADD_BOOKS, START_LOADING, OPEN_BOOK} from './actions'
+
+const initialState = {
+  books: [],
+  fetchingBooks: false,
+  openBook: ""
```

Comment:
Are you using this part of the state?
If not remove all code related to `openBook`.